scripts/AbinsModules/GeneralDFTProgram.py

```diff
@@ -0,0 +1,188 @@
+from mantid.kernel import logger
+
+# ABINS modules
+from IOmodule import IOmodule
+from KpointsData import KpointsData
+from AtomsData import AtomsDaTa
+from AbinsData import AbinsData
+import AbinsParameters
+import AbinsConstants
+
+
+# noinspection PyMethodMayBeStatic
+class GeneralDFTProgram(IOmodule):
```

Comment:
I would recommend GeneralDFTProgram is not inherited from IOmodule because you wouldn't naturally say GeneralDFTProgram is a child of IOmodule. Instead have IOmodule a component on its own (that has child classes like LoadCASTEP) and have IOmodule is component of GeneralDFTProgram